src/dmd/id.d

```diff
@@ -274,6 +274,7 @@ immutable Msgtable[] msgtable =
     { "opDispatch" },
     { "opDollar" },
     { "opUnary" },
+    { "opUnaryRight" }, // for `@disable`ing postincrement operators
```

Comment:
Why not allow disabling pre-increment operators as well? If this is for c++ binding than I see no reason to not kill 2 birds with one stone.

Reply:
> Why not allow disabling pre-increment operators as well?

That is already possible, or rather it is possible to disable increment and decrement operators entirely since the postfix for is rewritten to the prefix form.

> If this is for c++

This is for atomic types were the naive rewrite of postfix to prefix break atomicity. It just so happens to be for C++'s `std::atomic<T>`.